pkg/crc/cluster/status.go

```diff
@@ -0,0 +1,40 @@
+package cluster
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/code-ready/crc/pkg/crc/logging"
+	"github.com/code-ready/crc/pkg/crc/oc"
+)
+
+// WaitForClusterStable checks that the cluster is running a number of consecutive times
+func WaitForClusterStable(ocConfig oc.Config, monitoringEnabled bool) error {
+	startTime := time.Now()
+
+	retryDuration := 30 * time.Second
+	retryCount := 30
```

Comment:
So with this we are actually going to wait for around 900 sec before tell user that cluster is still in unstable state?

Reply:
Logs will tell him it's stuck before but yes, crc will be stuck 15min (this is what we have in the CI).

Reply:
I put 10min now

Reply:
if people are tired of waiting, they will break out. something in that case to capture and send as telemetry